src/Curry.jl

```diff
@@ -83,7 +84,8 @@ bind(f, args...) = Bind(f, args)
 macro bind(ex)
     ex.head == :call || error()
     # `ex.args[1]` is the function and `ex.args[2:end]` are the positional arguments
-    return :(bind($(map(esc, ex.args)...)))
+    TODO_HANDLE_KW = ()
```

Comment:
Does this serve the same purpose as a `TODO`comment?

Reply:
yeah sorry it is a leftover from some other thing I tried.